src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+
+        if (user != null) {
+            return user.getPassword().equals(password);
+        }
+
         return false;
```

Comment:
You can simplify it by making one return statement:
`return user != null && user.getPassword().equals(password);`